src/hooks/useGetTokens.ts

```diff
@@ -244,7 +243,7 @@ export type TokenData = {
 }
 
 function getPercentageDifference(a: number, b: number): number {
-  return b ? ((a - b) / a) * 100 : 0
+  return a && b ? ((a - b) / a) * 100 : 0
```

Comment:
If `b` is `0`, it'll return `0` as opposed to `100`

Is this the desired behaviour?

Reply:
if we do that then the % will be misleading, lets say the current (a) eth price is 2.000 but we don't have data for the last 7 days (b), would be ok to show 100% change on the Last 7 days column?

Reply:
Then you don't need that `if` on top, and can remove the requirement for `b` to be truthy:

```suggestion
  return a ? ((a - b) / a) * 100 : 0
```

If `b` is `0`, a/a * 100 => 100

Reply:
🤦‍♂️this is what happens when you don't sleep enough. Thanks